ios/keyman/Keyman/Keyman/InfoViewController/InfoViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  InfoViewController.swift
+//  Keyman
+//
+//  Created by Gabriel Wong on 2017-09-05.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+import UIKit
+
+class InfoViewController: UIViewController, UIWebViewDelegate {
+  @IBOutlet var webView: UIWebView!
+
+  var networkReachable: Reachability?
+
```

Comment:
If my understanding of [Swift initialization](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Initialization.html#//apple_ref/doc/uid/TP40014097-CH18-ID213) (see section "Automatic Initializer Inheritance") is correct, the reason we don't see an analogue to the old initializer of the class is because Swift automatically creates and manages its call for us?

Reply:
The old initializer merely called `super.init()` so I just removed it. A translation into Swift would be something like

```swift
init(nibName: String, bundle: Bundle) {
  super.init(nibName: nibName, bundle: bundle)
}
```

From my understanding of Swift, if a class does not declare an initializer, it inherits all initializers from its superclass. If a class declares any initializer, it doesn't inherit any initializer from its superclass and any required initializers must be explicitly defined.

So in this class, the call to `super.init()` is implicit.